pypeit/spectrographs/keck_kcwi.py

```diff
@@ -1031,46 +1032,164 @@ def get_rawimage(self, raw_file, det):
         # Return
         return detpar, raw_img, hdu, exptime, rawdatasec_img, oscansec_img
 
-    def scattered_light(self, frame, binning):
+    def scattered_light_model(self, param, img, kernel='gaussian'):
```

Comment:
I tend to agree... it would be good to have a default model that can be used for all spectrographs, but I think this will require spectrograph-specific coding to do it well.